test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -55,6 +57,48 @@ public void TestPfiRegressionOnDenseFeatures()
             Done();
         }
 
+        /// <summary>
+        /// Test PFI Regression for Dense Features with a model loaded from disk
+        /// </summary>
+        [Fact]
+        public void TestPfiRegressionOnDenseFeaturesWithLoadedModel()
+        {
+            var data = GetDenseDataset();
+            var model = ML.Regression.Trainers.OnlineGradientDescent().Fit(data);
+
+            var modelAndSchemaPath = GetOutputPath("TestPfiRegressionOnDenseFeatures.zip");
+            ML.Model.Save(model, data.Schema, modelAndSchemaPath);
+
+            ITransformer loadedModel;
+            using (var fs = File.OpenRead(modelAndSchemaPath))
```

Comment:
You aren't using `fs` here. This can be removed.

Reply:
Fixed in my last commit.